common/src/main/java/com/revenuecat/purchases/common/Backend.kt

```diff
@@ -238,6 +238,18 @@ class Backend(
         onError: (PurchasesError) -> Unit
     ) {
         val path = "/subscribers/" + encode(appUserID) + "/offerings"
+        val cacheKey = synchronized(this@Backend) {
+            // If there is any enqueued `postReceiptData` we don't want this new
+            // `getOfferings` to share the same cache key.
+            // If it did, future `getOfferings` would receive a cached value
+            // instead of an up-to-date `CustomerInfo` after those post receipt operations finish.
```

Comment:
> instead of an up-to-date `CustomerInfo` after those post receipt operations finish.
I am confused about this. This is fetching offerings, not `CustomerInfo` 🤔  Was this fix meant to be done on `getCustomerInfo`? `getOfferings` is not dependent on the receipt so I am not sure I understand

Reply:
Because I'm an idiot 🤦‍♂️ I don't know what I was thinking.
I just fixed this.